app/controllers/dataset_file_schemas_controller.rb

```diff
@@ -4,6 +4,7 @@ class DatasetFileSchemasController < ApplicationController
 
   def index
     @dataset_file_schemas = DatasetFileSchema.where(user: current_user).order(created_at: :desc)
+    @public_schemas = DatasetFileSchema.where(restricted: false).order(created_at: :desc)
```

Comment:
does this mean schemas already in the database will have the restricted flag disabled by default?